src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentManagementBackedDocumentsService.java

```diff
@@ -23,8 +21,7 @@
 import static uk.gov.hmcts.cmc.claimstore.utils.DocumentNameUtils.buildSealedClaimFileBaseName;
 import static uk.gov.hmcts.cmc.domain.models.ClaimDocumentType.SEALED_CLAIM;
 
-@Qualifier("DocumentManagementBackedDocumentsService")
-@Service
+@Service("DocumentsService")
```

Comment:
same here please

Reply:
sorry didn't get you.. it is already done isn't it